### PR TITLE
gitless-0.8.4

### PR DESCRIPTION
**Pull request by kontrafiktion**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

[gitless](http://gitless.com/) is a CLI that wraps git and provides an alternative
workflow without the concept of a staging area.

Closes #5519

### CODE REVIEW

File: Formula/gitless.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    url "https://files.pythonhosted.org/packages/b3/b2/238e2590826bfdd113244a40d9d3eb26918bd798fc187e2360a8367068db/six-1.10.0.tar.gz"
+    sha256 "105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/gl", "--help"
```
Comment (MikeMcQuaid) [inline Formula/gitless.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (MikeMcQuaid) [inline Formula/gitless.rb]: Perhaps in this case: `git init` a repo and perform some commits.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/gitless.rb (hunk 1 above)
Comment (kontrafiktion) [inline Formula/gitless.rb]: I "copied" the tests from git.rb and adapted them. 
I added "git" to the list of dependencies, but `brew audit` told me not to do so

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `gitless 0.8.4 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**